src/pages/Cart/Cart.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import OrderItems from './OrderItems/OrderItems';
+import './Cart.scss';
+
+class Cart extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      items: [],
+      totalPrice: 0,
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/item.json')
+      .then(res => res.json())
+      .then(res => {
+        this.setState({
+          items: res.items,
+        });
+      });
+  }
+
+  plusBtn = e => {
+    const { items } = this.state;
+    const changeList = [...items];
+    changeList.map(el => {
+      if (Number(e.target.name) === el.id) {
+        el.quantity = el.quantity + 1;
+      }
+    });
+    this.setState({
+      items: changeList,
+    });
+  };
+
+  minusBtn = e => {
+    const { items } = this.state;
+    const changeList = [...items];
+    changeList.map(el => {
+      if (Number(e.target.name) === el.id) {
+        el.quantity = el.quantity - 1;
+        if (el.quantity < 1) {
+          el.quantity = 1;
+        }
+      }
+    });
+    this.setState({
+      items: changeList,
+    });
+  };
+
+  test = data => {
+    let result = '';
+    result = data;
+    console.log('totalPrice=', result);
+    this.setState({
+      totalPrice: result,
+    });
+  };
+
+  render() {
+    const { items, totalPrice } = this.state;
+    const { plusBtn, minusBtn } = this;
+    let shippingFee = 2500;
+    if (totalPrice > 20000 || totalPrice === 0) {
+      shippingFee = 0;
+    }
```

Comment:
```suggestion
    const shippingFee =  totalPrice > 20000 || totalPrice === 0 ? 0 : 2500;
```
- 가급적 Const로 선언해주시고 이 경우에는 삼항 연산자로 처리해줄 수 있겠네요